packages/react/src/components/TooltipDefinition/TooltipDefinition.js

```diff
@@ -43,10 +46,17 @@ const TooltipDefinition = ({
       [`${prefix}--tooltip--${direction}`]: direction,
       [`${prefix}--tooltip--align-${align}`]: align,
       [`${prefix}--tooltip--hidden`]: !allowTooltipVisibility,
+      [`${prefix}--tooltip--visible`]: tooltipVisible,
     }
   );
+  const debounceTooltipVisible = debounce(() => setTooltipVisible(false), 100);
```

Comment:
Do we need to debounce the exit?

Reply:
the 100ms delay value is arbitrary but debouncing the exit is what resolves the original issue. afaik there is no guidance on the delay since the guidelines are still WIP https://github.com/w3c/aria-practices/issues/127 https://github.com/w3c/aria-practices/issues/128